guildwars2/misc.py

```diff
@@ -11,14 +11,29 @@
 class MiscMixin:
     @commands.command(aliases=["gw2wiki"])
     async def wiki(self, ctx, *, search):
-        """Search the Guild wars 2 wiki"""
+        """Search the Guild wars 2 wiki, append a language to search localized
```

Comment:
Keep the short help command the same - move the detailed part into detailed help